marshaller/marshaller.go

```diff
@@ -109,7 +109,7 @@ func New(shutdownHandler shutdown.ShutdownHandler,
 	statsChan chan stats.Stat,
 	noMarshalOldValue bool) Marshaller {
 
-	outputChan := make(chan *MarshalledMessage, 1000)
```

Comment:
This didn't have the change I was hoping for after more thorough testing so I'm rolling it back.